deploy_tools/cli.py

```diff
@@ -348,15 +354,22 @@ def call(
     click.echo(result)
 
 
-@main.command(short_help="Generate a new encrypted keystore file")
+@main.command(
+    short_help="Generates an encrypted keystore file. Creates a new private key of none is provided."
```

Comment:
`Creates a new account if no private key is provided.`

Reply:
:heavy_check_mark: